spartan/scripts/test_spartan.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# NOTE! currently does not work with 'cannot validate constraint' on tx simulate via pxe
```

Comment:
worth including here if it does not work?

Reply:
it turns out to just be mismatched contract versions, I would say so

Reply:
Removed the warning